chart/values.yaml

```diff
@@ -1830,6 +1830,14 @@ config:
     worker_container_repository: '{{ .Values.images.airflow.repository | default .Values.defaultAirflowRepository }}'
     worker_container_tag: '{{ .Values.images.airflow.tag | default .Values.defaultAirflowTag }}'
     multi_namespace_mode: '{{ ternary "True" "False" .Values.multiNamespaceMode }}'
+  kubernetes_executor:
+    namespace: '{{ .Release.Namespace }}'
+    airflow_configmap: '{{ include "airflow_config" . }}'
+    airflow_local_settings_configmap: '{{ include "airflow_config" . }}'
```

Comment:
```suggestion
```

These 2 are for 1.10 and earlier versions, yeah? We definitely don't need to duplicate those!

Reply:
I removes those 2 properties and added comments.

Reply:
Thanks for the help. Hopefully, I updated the PR appropriately.